w3-hw1-vy/cypress/e2e/studentRegistrationForm.cy.js

```diff
@@ -0,0 +1,211 @@
+    cy.get("@account").then((account) => {
+      registrationPage
+        .typeFirstName(account.user[0].firstName)
+        .typeLastName(account.user[0].lastName)
+        .chooseGender(account.user[0].gender)
+        .typeUserNumber(account.user[0].mobile)
+        .chooseDateOfBirth(account.user[0].day, account.user[0].month, account.user[0].year)
+        .clickSubmit();
+
+      confirmDialog
+        .isNameCorrect(account.user[0].firstName, account.user[0].lastName)
+        .isGenderCorrect(account.user[0].gender)
+        .isMobileCorrect(account.user[0].mobile)
+        .isDOBCorrect(account.user[0].day, account.user[0].month, account.user[0].year)
+
+    });
+  });
+});
+
+
+describe("The user can not sign up", () => {
+  beforeEach(() => {
+    cy.fixture("account").as("account");
+    cy.visit(Cypress.env("registrationForm"));
+  });
+
+  it("TC-SIGNUP-5: The user can not sign up with all empty information", () => {
+    cy.get("@account").then((account) => {
+      registrationPage
+        .clickSubmit();
+
+      registrationPage
+        .isFirstNameValid(false)
+        .isLastNameValid(false)
+        .isGenderValid(false)
+        .isMobileValid(false)
+        .isDOBValid(true);
+    });
+  });
+
+  it("TC-SIGNUP-7: The registration page does not accept leading or trailing spaces in FirstName and LastName", () => {
+    cy.get("@account").then((account) => {
+      registrationPage
+        .typeFirstName(account.user[2].firstName)
+        .typeLastName(account.user[2].lastName)
+        .typeUserEmail(account.user[2].email)
+        .chooseGender(account.user[2].gender)
+        .typeUserNumber(account.user[2].mobile)
+        .chooseDateOfBirth(account.user[2].day, account.user[2].month, account.user[2].year)
+        .chooseSubject(account.user[2].subjects)
+        .chooseHobbies(account.user[2].hobbies)
+        .choosePicture(account.user[2].picture)
+        .typeCurrentAddress(account.user[2].currentAddress)
+        .chooseState(account.user[2].state)
+        .chooseCity(account.user[2].city)
+        .clickSubmit();
+
+      confirmDialog
+        .isNameCorrect(account.user[2].firstName, account.user[2].lastName)
+        .isEmailCorrect(account.user[2].email)
+        .isGenderCorrect(account.user[2].gender)
+        .isMobileCorrect(account.user[2].mobile)
+        .isDOBCorrect(account.user[2].day, account.user[2].month, account.user[2].year)
+        .isPictureCorrect(account.user[2].picture)
+        .isSubjectsCorrect(account.user[2].subjects)
+        .isHobbiesCorrect(account.user[2].hobbies)
+        .isAddressCorrect(account.user[2].currentAddress)
+        .isStateCorrect(account.user[2].state, account.user[2].city);
+    });
+  });
+
+  it("TC-SIGNUP-8: The user can not sign up with:\n - Email includes multiple '@'\n - Mobile number has alphabet characters\n - Picture has invalid file format", () => {
```

Comment:
In automation E2E you can not cover all case-by-case, you should store your invalid data in an array and loop through it for invalid case, and name the case `Cannot submit with invalid email` or something like that